<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ManoHarsha" failures="2" tests="4" name="Surefire test" time="12.549" errors="0" timestamp="06 Aug 2023 12:28:15 GMT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.58">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.LoginSteps.i_verify_title_should_be(LoginSteps.java:23)
at ✽.I verify title should be "Login - My Show"(file:///C:/Users/manoh/CucumberProjects/cucumber1/cucumberTestNGPOM/src/test/resources/parallel/loginPage.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.621"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.68"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.185">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;4&amp;gt; but was:&amp;lt;5&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<4> but was:<5>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at parallel.AccountsPageSteps.account_selection_count_is(AccountsPageSteps.java:75)
at ✽.account selection count is 4(file:///C:/Users/manoh/CucumberProjects/cucumber1/cucumberTestNGPOM/src/test/resources/parallel/AccountPage.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Surefire test -->
